torch/_torch_docs.py

```diff
@@ -4362,3 +4362,47 @@
     [torch.FloatTensor of size 5]
 
 """)
+
+add_docstr(torch._C.btrifact,
+           """
+btrifact(A) -> Tensor
+
+Batch LU factorization.
+
+Returns a tuple containing the LU factorization, pivots, and pivot format,
+and info if the factorizations succeeded across the minibatch.
+The error codes are from cublas' dgetrf and a non-zero value
```

Comment:
No, I've clarified this in https://github.com/pytorch/pytorch/pull/903/commits/55b963b97189b6690ff5709d538ab31298b89fa5